playwright/e2e/request-baseline.spec.ts

```
@@ -0,0 +1,195 @@
+ *
+ * Context: Akamai DDoS protection triggered on console.redhat.com due to
+ * excessive Kessel API traffic (/api/kessel/v1beta2/checkself and
+ * /checkselfbulk). The kessel path was excluded from rate limiting as a fix.
+ * This test prevents regression by asserting request counts stay within
+ * safe thresholds.
+ *
+ * Run:
+ *   E2E_USER=you@redhat.com E2E_PASSWORD=... npx playwright test request-baseline
+ */
+
+import { test, expect } from '../setup/test-setup';
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)